lib/commands/general.js

```diff
@@ -87,6 +88,18 @@ commands.setUrl = async function (url) {
   await this.adb.startUri(url, this.opts.appPackage);
 };
 
+commands.mobileDeepLink = async function (opts = {}) {
+  const {url, package:pkg} = opts;
```

Comment:
what is the point to rename pkg->package?

Reply:
Package is a reserved word and breaks the linter